src/components/views/settings/leave.tsx

```diff
@@ -71,7 +70,11 @@ export const Leave = ({navigation}) => {
         <Spacing s={32} />
         <DataProtectionLink />
         <Spacing s={32} />
-        <Markdown style={{}}>{t('leave:summary')}</Markdown>
+        <Markdown style={{}}>
+          {Platform.OS === 'ios'
```

Comment:
Personally would have done:

```
t(`leave:summary.${Platform.OS}`)
``` 

and accommodated in the lang file:

```
leave: {
    summary: {
        android: '',
        ios: ''
    }
}
```

Reply:
👍